velox/functions/lib/Re2Functions.cpp

```diff
@@ -427,6 +588,14 @@ class LikeConstantPattern final : public VectorFunction {
  private:
   RE2 re_;
   bool validPattern_;
+  bool isConstPattern_;
```

Comment:
Why is this variable named with is and the others with has ? We should make the usage uniform.